frontend/src/constants/statuses.js

```diff
@@ -1,12 +1,19 @@
-export const statuses = {
-  new: 'New',
-  draft: 'Draft',
-  deleted: 'Deleted',
-  sent: 'Sent',
-  submitted: 'Submitted',
-  recommended: 'Recommended',
-  unregistered: 'Unregistered',
-  registered: 'Registered',
-  suspended: 'Suspended',
-  canceled: 'Canceled'
+export const TRANSFER_STATUS = {
```

Comment:
Let's update these to be plural so we know they are options lists. TRANSFER_STATUSES, ORGANIZATION_STATUSES

export const ORGANIZATION_STATUSES = {
  UNREGISTERED: 'Unregistered',
  REGISTERED: 'Registered',
  SUSPENDED: 'Suspended',
  CANCELED: 'Canceled'
};

// Function to get all organization statuses as an array
function getAllOrganizationStatuses() {
  return Object.values(ORGANIZATION_STATUSES);
}

// Usage
const statusArr = getAllOrganizationStatuses();

Reply:
updated as per comments.